net_utils.js

```diff
@@ -187,10 +180,59 @@ exports.is_ip_in_str = function(ip, str) {
     return false;
 };
 
-exports.is_rfc1918 = function (ip) {
-    return (net.isIPv4(ip) && re_private_ipv4.test(ip));
+exports.is_private_ipv4 = function (ip) {
+
+    // RFC 1918, reserved as "private" IP space
+    if (/^10\./.test(ip)) return true;          // 10/8
```

Comment:
I just benchmarked - and this:

````
var re_1 = /^10\./;
var re_2 = /^192\.168\./;
var re_3 = /^172\.(1[6-9]|2[0-9]|3[01])\./;

exports.is_private_ipv4 = function (ip) {
    if (re_1.test(ip) || re_2.test(ip) || re_3.test(ip)) return true;
    return false;
}
````

And as you would expect - this Is always faster than making V8 compile the regexps every call as you're doing here.  Which is why the regexps were compiled on the previous version.